staging/src/k8s.io/apiserver/pkg/admission/interfaces.go

```diff
@@ -53,22 +53,22 @@ type Attributes interface {
 
 // Interface is an abstract, pluggable interface for Admission Control decisions.
 type Interface interface {
-	// Admit makes an admission decision based on the request attributes
-	Admit(a Attributes) (err error)
-
 	// Handles returns true if this admission controller can handle the given operation
 	// where operation can be one of CREATE, UPDATE, DELETE, or CONNECT
 	Handles(operation Operation) bool
 }
 
+type MutationInterface interface {
+	Interface
+
+	// Admit makes an admission decision based on the request attributes
+	Admit(a Attributes) (err error)
```

Comment:
`Revise`?

Reply:
> Revise?

Perhaps in a future pull.  Keeping this the same for now keeps this pull smaller and gives a slightly more orderly transition.

Reply:
> Revise?
> Perhaps in a future pull. Keeping this the same for now keeps this pull smaller and gives a slightly more orderly transition.

Oh, and I hadn't heard of `Revise` until a few days ago, so that'll be worth a conversation if you want to change the name.  That's a rarely used word and reminds me of Derek's "confide". :)